### PR TITLE
Mmonteith/autocomplete

### PR DESCRIPTION
**Pull request by mcheung-nhs**
Cleaned up version of feature/autocomplete branch

## Description

The Header component search autocomplete is now using the GOV.UK accessible autocomplete so that we can remove the Typeahead script which requires jQuery.

This is the only component within the Frontend library that requires jQuery, so we can now move away from using jQuery. This will improve performance as we won't have to load in a (large) third party library.

JavaScript bundle before: 53kb + 87 KB jQuery
JavaScript bundle after: 40kb

### CODE REVIEW

File: packages/components/header/autocomplete.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    return svgIcon + resultsHref;
+  }
+
+  const defaultConfig = {
+    element: document.querySelector('#autocomplete-container'),
+    id: id,
+    minLength: 2,
+    placeholder: fallbackInputElement.placeholder,
+    // TO DO:
```
Comment (AdamChrimes) [inline packages/components/header/autocomplete.js]: @mikemonteith there are a couple of 'TO DO' comments in this JavaScript file, do you know if we need to actually do anything or is it fine without? cc: @sabi98

Comment (mikemonteith) [reply packages/components/header/autocomplete.js]: I think in this case we should just delete these 3 commented lines.

Comment (sabi98) [reply packages/components/header/autocomplete.js]: Can remove lines 61 to 63;
  ```
// TO DO:
// cssNamespace: ??
// displayMenu: 'overlay', this need enable but currently breaks.
```

Comment (mcheung-nhs) [reply packages/components/header/autocomplete.js]: Comments removed,

**Pull request merged**